src/viur/core/email.py

```diff
@@ -142,14 +143,14 @@ def normalize_to_list(value: Union[None, Any, List[Any], Callable[[], List]]) ->
 def sendEMail(*,
               tpl: str = None,
               stringTemplate: str = None,
-              skel: Union[None, Dict, "SkeletonInstance", List["SkeletonInstance"]] = None,
+              skel: None | dict | SkeletonInstance | list[SkeletonInstance] = None,
```

Comment:
```py
              skel: None | dict | "SkeletonInstance" | list["SkeletonInstance"] = None,  # causes: TypeError: unsupported operand type(s) for |: 'types.UnionType' and 'str'
```
would cause 

> TypeError: unsupported operand type(s) for |: 'types.UnionType' and 'str'

so we have to use 

```suggestion
              skel: t.Union[None, dict, "SkeletonInstance", list["SkeletonInstance"]] = None,
```